src/content/contributions/add-ons/add-ons.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Add-ons
+---
+
+import { CheckmarkFilled16 } from '@carbon/icons-react';
+
+<AnchorLinks>
+
+- [Add-on repositories](#add-on-repositories)
+- [Contributing to add-ons](#contributing-to-add-ons)
+- [Ownership and maintenance](#ownership-and-maintenance)
+
+</AnchorLinks>
+
+## Add-on repositories
+
+Add-on repositories, or repos, contain components built for a specific product or experience. The repos are built on top of the core Carbon repo, and enable teams to create their own custom components that follow Carbon's visual style and guidelines.
+
+If your team is using Carbon and needs components specific to your product or industry, host the custom components in an add-on repo. We have created a [React boilerplate](https://github.com/carbon-design-system/carbon-addons-boilerplate-react) and a [Vanilla boilerplate](https://github.com/carbon-design-system/carbon-boilerplate) that you can clone to get started.
```

Comment:
This usage of repo seems perfect, however, do we want to call out the boilerplates? Their super outdated and I would not recommend them personally.

Reply:
If you don't recommend calling out the boilerplates we most definitely won't.